### PR TITLE
add some graphics

### PR DESCRIPTION
**Pull request by Nils-Magnus**

### CODE REVIEW

**Comment by tischrei** [comment]
recheck

File: doc/source/additional/sd-manual.rst [hunk 1]
```diff
@@ -2,55 +2,84 @@
...
+Services are categorized into sections such as Compute, Network, or
+Storage. Each service comes with an icon, also known as a
+semaphore. Click on the icons to find out more about the
+incident. This way you will find latest updates issued by the service
+manager for the ongoing resolution progress, further investigations,
+mitigation plans, and timelines. The service manager announces when a
+next update is planned.
 
 - **Green:** Service fully operational
+  .. figure:: /_static/images/sd/1-operational.png
```
Comment (tischrei) [inline doc/source/additional/sd-manual.rst:48]: all images are not rendered

**Review by tischrei — CHANGES_REQUESTED** [review]

**Comment by Nils-Magnus** [comment]
recheck

**Comment by tischrei** [comment]
@Nils-Magnus we will not have the content here. Please transfer the content to: https://gitea.eco.tsi-dev.otc-service.com/docs/status-dashboard

**Pull request closed without merging**